databricks_sql_connector_core/src/databricks_sql_connector_core/sql/auth/retry.py

```diff
@@ -14,7 +14,7 @@
 from urllib3 import Retry
 from urllib3.util.retry import RequestHistory
 
-from databricks.sql.exc import (
+from databricks_sql_connector_core.sql.exc import (
```

Comment:
please do not change the namespace and also keep `databricks` namespace as the root.